web/src/components/tasks/business-structure/BusinessStructureTask.tsx

```diff
@@ -91,6 +92,10 @@ export const BusinessStructureTask = (props: Props): ReactElement => {
       })
       .queueTaskProgress({ [props.task.id]: "NOT_STARTED" });
 
+    if (business) {
+      await updateQueue.queueFormationFormData(business?.formationData.formationFormData).update();
+    }
+
```

Comment:
Same question.

Reply:
This is necessary, because the address no longer persists when the user interacts with the business structure task.